core-ui/src/components/Extensibility/components-form/StringRenderer.js

```diff
@@ -15,7 +18,8 @@ export function StringRenderer({
   placeholder,
   ...props
 }) {
-  const { tFromStoreKeys, t: tExt } = useGetTranslation();
+  const { tFromStoreKeys } = useGetTranslation();
+  const schemaPlaceholder = schema.get('schemaPlaceholder');
```

Comment:
In the rest of the files, we are "getting" `'placeholder'` so I think we need to have one convention.